docs/openapi.json

```diff
@@ -384,6 +384,18 @@
                   "title": "Default workspace",
                   "value": "/usr/share"
                 },
+                "kubernetes_max_memory_limit": {
+                  "title": "Maximum allowed memory limit for Kubernetes jobs",
+                  "value": "4Gi"
```

Comment:
10Gi

It'll be good to put "realistic" numbers following `reana.cern.ch` typical deployment, so that users that happen to see these examples won't think it's the REANA defaults.